easybuild/easyconfigs/s/Sambamba/Sambamba-0.8.0-GCC-10.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'MakeCp'
+
+name = 'Sambamba'
+version = '0.8.0'
+
+homepage = 'https://lomereiter.github.io/sambamba/'
+description = """Sambamba is a high performance modern robust and fast tool
+ (and library), written in the D programming language, for working with SAM
+ and BAM files. Current functionality is an important subset of samtools
+ functionality, including view, index, sort, markdup, and depth."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/biod/sambamba/archive/']
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/biod',
+        'repo_name': 'sambamba',
+        'commit': 'f898046',
+        'recursive': True,
+    }
+}]
+checksums = [None]
+
+builddependencies = [
+    ('LDC', '1.26.0'),
+    ('Python', '3.9.5'),
```

Comment:
In `0.8.2`, they [no longer bundle `lz4` sources](https://github.com/biod/sambamba/releases/tag/v0.8.2), so `('lz4', '1.9.3'),` is needs to be added here.